packages/babel-parser/test/fixtures/experimental/import-assertions/invalid-export-without-from/options.json

```diff
@@ -0,0 +1,9 @@
+{
+  "plugins": [
+    [
+      "importAssertions"
+    ]
+  ],
+  "sourceType": "module",
+  "throws": "Unexpected token, expected \";\" (2:15)"
```

Comment:
Wonder if we should point out a missing `from` in the error message given `assertions`? (Might be better DX)

Reply:
I think it is okay here because when `from` is missing
```
export { someExport } asserts { type: "json" }
```
it would first throw
```
"someExport" is not defined
```
If users have already provided `someExport`, it is unlikely `someExport` could be exported from another module and thus they should consider remove the assertions.